contracts/Comptroller.sol

```diff
@@ -1229,9 +1196,8 @@ contract Comptroller is ComptrollerV5Storage, ComptrollerInterface, ComptrollerE
         for (uint i = 0; i < cTokens.length; i++) {
             CToken cToken = cTokens[i];
             require(markets[address(cToken)].isListed, "market must be listed");
-            if (borrowers == true) {
+            if (borrowers == true && address(cToken) != getXinvAddress()) {
                 Exp memory borrowIndex = Exp({mantissa: cToken.borrowIndex()});
-                updateCompBorrowIndex(address(cToken), borrowIndex);
```

Comment:
Why not remove this code path?

Reply:
In order to allow for retroactive distribution of borrowing rewards